x/tokenomics/keeper/update_relay_mining_difficulty.go

```diff
@@ -123,37 +122,43 @@ func ComputeNewDifficultyTargetHash(targetNumRelays, newRelaysEma uint64) []byte
 	// the actual on-chain relays, so we don't need to scale to anything above
 	// the default.
 	if targetNumRelays > newRelaysEma {
-		return defaultDifficultyTargetHash()
+		return prooftypes.DefaultRelayDifficultyTargetHash
 	}
 
-	log2 := func(x float64) float64 {
-		return math.Log(x) / math.Ln2
-	}
+	// Calculate the proportion of target relays to the new EMA
+	ratio := float64(targetNumRelays) / float64(newRelaysEma)
+
+	// Compute the new target hash by scaling the default target hash based on the ratio
+	newTargetHash := scaleDifficultyTargetHash(prooftypes.DefaultRelayDifficultyTargetHash, ratio)
```

Comment:
I'm still pretty confused by what's happening here.

1. What kind of distribution does this result in?

2. If the number of relays is VERY high (e.g. 10B per session), but is constant (has been that way for a long time), would we make the difficulty low because you're always scaling the default?

Reply:
Apologies, I think this should actually be scaling the current target hash rather than the default. :sweat_smile: 

My understanding is that `newRelaysEma` represents on-chain/mined/payable relays, which means that given a constant high number of relays, as target hash adjustments are made, `T/R` should eventually approach `1`. This means that we need to scale the current target hash proportionally and it should stabilize as `T/R` approaches `1`.